src/stores/process.ts

```diff
@@ -168,14 +168,36 @@ export function createCommandFactory<T, P extends object = DefaultPayload>(): Co
 export type Commands<T = any, P extends object = DefaultPayload> = (Command<T, P>[] | Command<T, P>)[];
 
 const processMap = new Map();
-const valueSymbol = Symbol('value');
+export const valueSymbol = Symbol('value');
 
 export function getProcess(id: string) {
 	return processMap.get(id);
 }
 
 const proxyError = 'State updates are not available on legacy browsers';
 
+function removeProxies(value: any) {
```

Comment:
This could be premature, but I tend to avoid using `map` and `reduces` on what could be hot paths with a significant number of keys/items. Would it be possible to use a standard loop? Does the it need to be immutable? Or would it be fine to mutate the existing object? I was thinking something like this, which I think could be made immutable if required.

```ts
	if (typeof value === 'object' && value !== null) {
		if (value[valueSymbol]) {
			value = value[valueSymbol];
		}
		const keys = Object.keys(value);
		for (let i = 0; i < keys.length; i++) {
			value[keys[i]] = removeProxies(value[keys[i]]);
		}
	}
	return value;
```

Reply:
This object could be a user created object that was passed in, so we would not want to mutate it. I can use a loop to build the object though.

Reply:
👍 Updated to use a standard loop without mutating the arg